.changeset/seven-ghosts-try.md

```diff
@@ -3,6 +3,6 @@
 "@fluid-experimental/tree": major
```

Comment:
Why is it this?

Reply:
The original PR made changes to tree: https://github.com/microsoft/FluidFramework/pull/17210/files

Reply:
Ok yeah I see that.  I think that original changeset should have higlighted container-loader not tree.  This is the core of that change as far as I can see:  https://github.com/microsoft/FluidFramework/pull/17210/files#diff-c94f4bb2aaac11f9fc45cb8bfd7c09d091d54e38e437b727ac1244f1e8d5cb6c

I'd change it (cc @scottn12 )

Reply:
Removed the entry for tree.